portfolio/src/main/webapp/maps.js

```diff
@@ -72,3 +70,12 @@ function placeMarker(query, map) {
   });
 }
 
+/**
+ * Fetches the place queries from the Java servlet and places markers.
+ */
+async function loadPlaceQueries(map) {
+  const response = await fetch('/map-data');
+  const placeData = await response.json();
+  placeData.forEach(place => placeMarker(place, map));
```

Comment:
Can you also close the other previously opened place marker info boxes when a user opens a new one?

Reply:
It might also be interesting to include a blurb about what you like about each place that displays with the place/address!

Reply:
+1 to closing the previously opened markers

Reply:
I think the closing previously opened markers should work now! The change has also been deployed. I'm planning on adding blurbs, but that will involve modifying the servlet and adding an extra class, so it will take longer to do.